recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domd-h3ulcb.cfg

```diff
@@ -33,7 +33,6 @@ dt_compatible = [ "renesas,h3ulcb", "renesas,r8a7795" ]
 # Clocks and regulators are needed and are in the device tree root,
```

Comment:
This patch does need explanation and rationale for the changes

Reply:
ok, will explain in details